mappings/net/minecraft/block/entity/DropperBlockEntity.mapping

```diff
@@ -1 +1,2 @@
 CLASS net/minecraft/class_3902 net/minecraft/block/entity/DropperBlockEntity
+	METHOD method_26975 initDropperDataFixer (Lnet/minecraft/class_2656;)V
```

Comment:
Why init`Dropper`DataFixer ? Just initDataFixer sounds good, u anyways would call it like `DropperBlockEntity.initDataFixer()` so that in itself should be enough to understand that the datafixer is for dropper

Reply:
conflicts

Reply:
Fixed myself